### PR TITLE
Updates of latest features in documentation

### PR DESCRIPTION
**Pull request by fviale**
- add improvements for the Dockerfile script engine
 - add public urls variables
 - add scheduler start state options (frozen, paused, stopped)
 - final fix of Generic information reference
 - several fixes in glossary

### CODE REVIEW

File: src/docs/Glossary.adoc [hunk 1]
```diff
@@ -33,37 +33,57 @@ Scheduler Web Interface:: ProActive component that provides a web interface to t
 [[_glossary_workflow_studio]]
 Workflow Studio:: ProActive component that provides a web interface for designing <<_glossary_workflow,*Workflows*>>.
 
+[[_glossary_automation_dashboard]]
+Automation Dashboard:: Centralized interface for the following web portals: <<_glossary_workflow_automation_portal,*Workflow Automation*>>, <<_glossary_catalog_portal,*Catalog*>>, <<_glossary_analytics_portal,*Analytics*>>, <<_glossary_job_planner_portal,*Job Planner*>>, <<_glossary_cloud_automation_portal,*Cloud Automation*>>, <<_glossary_cloud_watch_portal,*Cloud Watch*>> and <<_glossary_notification_portal,*Notifications*>>.
+
+[[_glossary_analytics]]
+Analytics::  ProActive service responsible to gather monitoring information for <<_glossary_job,*Jobs*>> and <<_glossary_proactive_node,*ProActive Nodes*>>
+
 [[_glossary_job_analytics]]
```
Comment (nebilbenmabrouk) [inline src/docs/Glossary.adoc:42]: Why we user **service** for some modules and **component** for others

File: src/docs/Glossary.adoc [hunk 2]
```diff
@@ -33,37 +33,57 @@ Scheduler Web Interface:: ProActive component that provides a web interface to t
...
+Notification Method:: Component of the <<_glossary_notification_service,*Notification Service*>> that defines how the user is notified of notifications.
 
 [[_glossary_cloud_automation_service]]
 Cloud Automation Service:: ProActive component that allows a user to easily manage any On-Demand Services (PaaS, IaaS and MaaS) with full Life-Cycle Management (create, deploy, pause, resume and terminate).
 
+[[_glossary_cloud_automation_portal]]
+Cloud Automation Portal:: Web interface of the <<_glossary_cloud_automation_service,*Cloud Automation service*>>.
+
+[[_glossary_workflow_automation_portal]]
+Workflow Automation Portal:: Web interface which provides an end-user portal to submit workflows, monitor their activity and recieve workflow results.
```
Comment (nebilbenmabrouk) [inline src/docs/Glossary.adoc:73]: monitor their activit**ies**

File: src/docs/Glossary.adoc [hunk 3]
```diff
@@ -33,37 +33,57 @@ Scheduler Web Interface:: ProActive component that provides a web interface to t
...
 
 [[_glossary_notification_third_party]]
-Third Party Notification::
-Notification method of the <<_glossary_notification_service,*Notification Service*>> that executes a script when a notification is triggered.
+Third Party Notification:: Notification method of the <<_glossary_notification_service,*Notification Service*>> that executes a script when a notification is triggered.
 
 [[_glossary_notification_methods]]
-Notification Method::
-Component of the <<_glossary_notification_service,*Notification Service*>> that defines how the user is notified of notifications.
+Notification Method:: Component of the <<_glossary_notification_service,*Notification Service*>> that defines how the user is notified of notifications.
```
Comment (nebilbenmabrouk) [inline src/docs/Glossary.adoc:64]: how the user **receives** notifications

File: src/docs/Glossary.adoc (hunk 1 above)
Comment (fviale) [reply src/docs/Glossary.adoc:42]: I thought I will receive such comment. I could switch to service eventually, but in some cases it does not apply.

File: src/docs/user/references/ScriptLanguageSupport.adoc [hunk 4]
```diff
@@ -506,40 +509,105 @@ RUN sleep 30
 The execution of this example will create an image using the Docker build command and start a container from this image using the specified commands (echo and sleep).
 At the end, the built image and the started container are deleted.
 
-* The Dockerfile task allows to set parameters to the `docker build` command with regard to the docker-build CLI https://docs.docker.com/engine/reference/builder/[reference].
+The created image default tag is `image_${PA_JOB_ID}t${PA_TASK_ID}` and the default container tag is `container_${PA_JOB_ID}t${PA_TASK_ID}`.
+
+==== Docker actions
+
+The default sequence of docker commands (*build*, *run*, *stop*, *rm*, *rmi*) can be modified to use the docker script engine in different configurations such as multiple workflow tasks.
+
```
Comment (aminelouati) [inline src/docs/user/references/ScriptLanguageSupport.adoc:517]: maybe add a picture for the workflow visualization

File: src/docs/Glossary.adoc [hunk 5]
```diff
@@ -33,37 +33,57 @@ Scheduler Web Interface:: ProActive component that provides a web interface to t
...
 
 [[_glossary_notification_subscription]]
-Notification Subscription::
-Object of the <<_glossary_notification_service,*Notification Service*>> used to manage and configure notification reception.
+Notification Subscription:: Object of the <<_glossary_notification_service,*Notification Service*>> used to manage and configure notification reception.
 
 [[_glossary_notification_third_party]]
-Third Party Notification::
-Notification method of the <<_glossary_notification_service,*Notification Service*>> that executes a script when a notification is triggered.
+Third Party Notification:: Notification method of the <<_glossary_notification_service,*Notification Service*>> that executes a script when a notification is triggered.
```
Comment (nebilbenmabrouk) [inline src/docs/Glossary.adoc:61]: We can append another sentence to better explain, e.g., :
This script is generally used to send notifications to a third party monitoring system in the customer environment.

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 6]
```diff
@@ -94,6 +94,12 @@ The following ProActive Scheduler components and services are started:
 
 The URLs of the Scheduler, Resource Manager, REST API and Web Interfaces are displayed in the output.
 
+The ProActive server can also be started in a specific state by using the command line options `--stopped`, `--paused` or `--frozen`.
+
+When one of these options is used, the scheduler will start in the provided state, which can be changed afterwards using the ProActive Scheduler Portal.
```
Comment (nebilbenmabrouk) [inline src/docs/admin/ProActiveAdminGuide.adoc:99]: the ProActive Scheduler **p**ortal

File: src/docs/user/references/ScriptLanguageSupport.adoc [hunk 7]
```diff
@@ -484,15 +484,18 @@ key *docker-compose-options-split-regex*. If you supply e.g. "!SPLIT!" as value,
 
 === Dockerfile
 
-The default behavior of the Dockerfile task is to build an image and then start a container instance from it. Once the execution is done, the container is stopped and the built image is deleted. The build, start, stop and, remove actions can be parametrized through advanced command line options which will be detailed later.
+==== Introduction
 
+The Dockerfile engine allows to define a docker image using the Dockerfile syntax, build this image and run a container instance from it.
+Once the execution is done, the container is stopped and the built image is deleted.
+The build, start, stop and, remove actions can be parametrized through advanced command line options which will be detailed later.
```
Comment (nebilbenmabrouk) [inline src/docs/user/references/ScriptLanguageSupport.adoc:491]: which will be detailed later in **section <ref>**.

File: src/docs/user/references/VariablesReference.adoc [hunk 8]
```diff
@@ -208,6 +208,41 @@ ProgressFile.setProgress(variables.get("PA_TASK_PROGRESS_FILE"), 50);
...
+| -
+
+| *Job Planner Rest URL*. The REST url of the job-planner service. The public REST url is also available (if the setting pa.job-planner.rest.public.url is defined in `PROACTIVE_HOME/config/scheduler/settings.ini`)
+| variables.get( "PA_JOB_PLANNER_REST_URL" ) or variables.get( "PA_JOB_PLANNER_REST_PUBLIC_URL" )
+| $variables_PA_JOB_PLANNER_REST_URL or $variables_PA_JOB_PLANNER_REST_PUBLIC_URL
+| ${PA_JOB_PLANNER_REST_URL} or ${PA_JOB_PLANNER_REST_PUBLIC_URL}
+| all scripts
+| -
+
+| *Notification Service Rest URL*. The REST url of the notification service. The public REST url is also available (if the setting pa.notification-service.rest.public.url is defined in `PROACTIVE_HOME/config/scheduler/settings.ini`)
```
Comment (nebilbenmabrouk) [inline src/docs/user/references/VariablesReference.adoc:239]: **pa.notification-service.rest.public.url** (better to put it in italic to differentiate it from the rest of the text)

**Review by nebilbenmabrouk — APPROVED** [review]

**Review by aminelouati — APPROVED** [review]

File: src/docs/Glossary.adoc [hunk 9]
```diff
@@ -33,37 +33,57 @@ Scheduler Web Interface:: ProActive component that provides a web interface to t
...
+
+[[_glossary_analytics]]
+Analytics::  ProActive service responsible to gather monitoring information for <<_glossary_job,*Jobs*>> and <<_glossary_proactive_node,*ProActive Nodes*>>
+
 [[_glossary_job_analytics]]
-Job Analytics:: ProActive component that provides an overview of executed workflows along with their input variables and results.
+Job Analytics:: ProActive component, part of the <<_glossary_analytics,*Analytics*>> service, that provides an overview of executed workflows along with their input variables and results.
+
+[[_glossary_node_gantt]]
+Node Gantt:: ProActive component, part of the <<_glossary_analytics,*Analytics*>> service, that provides an overview of <<_glossary_proactive_node,*ProActive Nodes*>> usage over time.
```
Comment (mboussaa) [inline src/docs/Glossary.adoc:46]: and how about job gantt?

File: src/docs/Glossary.adoc (hunk 3 above)
Comment (mboussaa) [reply src/docs/Glossary.adoc:64]: remove `of notifications` and u might add method examples

**Review by mboussaa — APPROVED** [review]

File: src/docs/Glossary.adoc (hunk 2 above)
Comment (fviale) [reply src/docs/Glossary.adoc:73]: plural is not necessary here. Several workflows, each as an activity, so I can say their activity.

**Pull request merged**